python/podio/test_utils.py

```diff
@@ -2,59 +2,5 @@
 """Utilities for python unittests"""
 
 import os
```

Comment:
Hmm this file only has like 3 lines after all the removals, I'm not sure if it's needed...

Reply:
I think I can clean this up in the course of #485